docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,268 @@
+
+There are few ways to convert dataset to Pascal VOC format:
+
+``` bash
+# export dataset into Pascal VOC format (classification) from existing project
+datum export -p <path/to/project> -f voc -o <path/to/export/dir> -- --tasks classification
+
+# converting to Pascal VOC format from other format
+datum convert -if imagenet -i <path/to/imagenet/dataset> \
+    -f voc -o <path/to/export/dir> \
+    -- --label_map voc --save-images
+```
+
+Argument `--tasks` allow to specify tasks for export dataset,
+by default Datumaro uses all tasks.
+Argument   `--label_map` allow to define user label map, for example
+
+``` bash
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+```
+
+Or you can use original Pascal VOC label map:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Datumaro functionality
+
+Datumaro supports filtering, transformation, merging etc. for all formats
+and for the Pascal VOC format in particular. Follow
+[user manual](../docs/user_manual.md)
+to get more information about these operations.
+
+There are few examples of using Datumaro operations to solve
+particular problems:
+
+- Example 1. Preparing Pascal VOC dataset for converting to Market-1501 dataset format.
+Market-1501 dataset only has a person class, marked with a bounding box.
+And to perform the conversion we could filter the Pascal VOC dataset.
+With Datumaro we can do it like this
+
+``` bash
+# create Datumaro project with Pascal VOC dataset
+datum import -o myproject -f voc -i <path/to/voc/dataset>
+
+# convert labeled shapes into bboxes
+datum transform -t shapes_to_boxes
+
+# keep only person class items
+datum filter -p myproject-shapes_to_boxes \
+    --mode items+annotations \
+    -e '/item/annotation[label="person"]' \
+    -o tmp_project
+
+# delete other labels from dataset
+datum transform -p tmp_project -o final_project \
+    -t remap_labels -- -l person:person --default delete
+```
+
+To make sure that the converting was succesful we can check the output project:
+
+```bash
+cd <path/to/final/project>
+datum info
+```
+
+- Example 2. By default Pascal VOC 2012 dataset includes only one train and one validation subsets.
+With Datumaro you can use cross-validation approach for the task:
+
+```bash
+# create Datumaro project with Pascal VOC 2012
+datum import -f voc -i ./VOC2012
+
+# spliting dataset into 3 parts
+datum transform -o iter1 -t random_split -- -s train1:.33 -s train2:.33 -s val:.34
+
+# train and validate the model ...
+# reorganize subsets for 2nd iteration
+datum transform -p iter1 -o iter2 -t map_subsets -- -s train2:val -s val:train2
+
+# train and validate the model ...
+# reorganize subsets for 3rd iteration
+datum transform -p iter2 -o iter3 -t map_subsets -- -s train1:val -s val:train1
+
+# train and validate the model ...
+```
+
+- Example 3. If you don`t need a variety of classes in Pascal VOC dataset,
```

Comment:
Let's remove e 2 and 3 from this section and move them into a separate tutorial file. Then add an example of a parsed dataset in the code. Keep them simple, but informative.